### PR TITLE
Adds checking for https://punctum-medico.de/onlinetermine/

### PR DESCRIPTION
**Pull request by jozr**

### CODE REVIEW

File: demon.js [hunk 1]
```diff
@@ -186,6 +186,56 @@ function observeImpfstoff() {
...
+          availabilities = false;
+        }, 60000);
+
+        notify();
+      });
+    })
+    .catch(error);
+  }
+
+  setTimeout(observePunctumMedico, rateLimit);
```
Comment (Pita) [inline demon.js:286]: I would move this to the first line of the function. If any error happens during the function execution it won't be rescheduled.

File: demon.js [hunk 2]
```diff
@@ -186,6 +186,56 @@ function observeImpfstoff() {
...
+        }
+
+        log("Punctum Medico success", appt);
+
+        availabilities = true;
+        setTimeout(function () {
+          availabilities = false;
+        }, 60000);
+
+        notify();
```
Comment (Pita) [inline demon.js:277]: Should probably also open a webpage in case of success, no?

**Review by Pita — COMMENTED** [review]
Thank you, very cool to have another provider. I'm ok merging this, just let me know real quick what you think of the comments I made

File: demon.js [hunk 3]
```diff
@@ -232,6 +282,9 @@ data.forEach(function (links) {
 // Comment back in to observe impfstoff.link for availabilities.
 // observeImpfstoff();
 
+// Comment back in order to observe Punctum Medico
+// observePunctumMedico();
```
Comment (mousemke) [inline demon.js]: assuming #3 merges, we should default that one on

Comment (Pita) [reply demon.js]: yes

**Comment by Pita** [comment]
I integrated your code manually as I migrated the code base to typescript to better be able to verify PRs. Thank you for this contribution!

**Comment by jozr** [comment]
@Pita Of course. And TypeScript is a great update. 🥇

May I suggest committing with original author signature for next time?

**Comment by Pita** [comment]
@all-contributors please add @jozr for code

**Pull request closed without merging**